roles/job_runner/tasks/main.yml

```diff
@@ -7,6 +7,20 @@
     name: "{{ lookup('env', 'ANSIBLEJOB_NAME') }}"
     namespace: "{{ lookup('env', 'ANSIBLEJOB_NAMESPACE') }}"
   register: ansible_job
+  when:
+    - lookup('env','TOWER_JOB_TEMPLATE_NAME') is defined
+    - lookup('env','TOWER_JOB_TEMPLATE_NAME') != ''
+
+- name: Read AnsibleWorkflow Specs
+  k8s_info:
+    kind: AnsibleWorkflow
+    api_version: tower.ansible.com/v1alpha1
+    name: "{{ lookup('env', 'ANSIBLEWORKFLOW_NAME') }}"
+    namespace: "{{ lookup('env', 'ANSIBLEWORKFLOW_NAMESPACE') }}"
+  register: ansible_workflow
```

Comment:
I don't think this is currently referenced anywhere.  @cooktheryan I think you need to reference it here: https://github.com/ansible/awx-resource-operator/pull/87/files#diff-72886cf7b8009753cc2039b0b997ee8b841ef4fefffe6f959e52b13f428779b9R6